scripts/build/options.js

```diff
@@ -2,9 +2,11 @@
 const minimist = require("minimist");
 const os = require("os");
 
+const ci = ["1", "true"].includes(process.env.CI);

```

Comment:
This gets set by the workflow or something, right?  It's not something _we_ set elsewhere?

Reply:
Correct; GHA sets `CI=1` or similar.